libs/sdk-common/src/lnurl/specs/withdraw.rs

```diff
@@ -34,9 +36,11 @@ pub async fn validate_lnurl_withdraw(
 
     // Send invoice to the LNURL-w endpoint via the callback
     let callback_url = build_withdraw_callback_url(&req_data, &invoice)?;
-    let callback_url = maybe_replace_host_with_mock_test_host(callback_url)?;
 
-    let withdraw_status = match get_parse_and_log_response(&callback_url, false).await {
+    let response = rest_client
+        .get_and_log_response(&callback_url, false)
+        .await?;
+    let withdraw_status = match parse_json(&response) {
```

Comment:
The timeout branch in this match statement is no longer correct, since a timeout would come from the `get_and_log_response` call above.